mw-webapp/src/logic/reportsTable/renderCellValue/renderCellItem.tsx

```diff
@@ -8,7 +8,7 @@ import {MentorComment} from "src/model/businessModel/MentorComment";
 import {PlanForNextPeriod} from "src/model/businessModel/PlanForNextPeriod";
 
 /**
- * Properties of {@link DayReport} that have type string[] for dynamic update
+ * Properties of DayReport model that have type string[] for dynamic update
```

Comment:
Why not to use @link

Reply:
Bacuse we can links only to those models that was imported in file. Otherwise this link doesn't work